react/utils/aspectRatioUtil.js

```diff
@@ -4,24 +4,27 @@ import { changeImageUrlSize } from './normalize'
  * For example, turns "3:4" into 1.333, so the image height will be
  * 1.333 times its width.
  */
-const parseAspectRatio = input => {
+const parseAspectRatio = (input) => {
   if (!input) {
     return null
   }
+
   if (typeof input === 'string') {
     if (input === 'auto') {
       return null
     }
+
     const separator = ':'
     const data = input.split(separator)
+
     if (data.length !== 2) {
       return null
     }
 
     const [width, height] = data
     const ratio = parseFloat(height) / parseFloat(width)
 
-    if (typeof ratio !== 'number' || isNaN(ratio)) {
+    if (typeof ratio !== 'number' || Number.isNaN(ratio)) {
```

Comment:
`Number.isNaN` will literally check if a value is `NaN` and if it's not a number